.build/parent-pom-template.xml

```diff
@@ -628,17 +629,17 @@
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-core</artifactId>
-        <version>3.1.5</version>
+        <version>${metrics.version}</version>
       </dependency>
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-jvm</artifactId>
-        <version>3.1.5</version>
+        <version>${metrics.version}</version>
       </dependency>
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-logback</artifactId>
-        <version>3.1.5</version>
+        <version>${metrics.version}</version>
```

Comment:
opinion: not a huge fan of properties like this when the use is all in one adjacent place.

Reply:
I did this in accordance with the documentation:
https://metrics.dropwizard.io/4.2.0/getting-started.html

> Note
> Make sure you have a metrics.version property declared in your POM with the current version, which is 4.2.0.

Reply:
I'm not sure they know what they are talking about.

What does 
```
grep -r  "metrics.version" ~/.m2/repository/io/dropwizard/metrics
```
give you?

Educate me please if I'm missing something, I see nothing in their parent pom, and we're not pulling in a bom.

Reply:
I don't know the reasoning covered behind the message, but I understand it as follows: if a user is using the cassandra parent and wants to use additional modules provided by the dropwizard metrics project he/she should refer to the {metrics.version} inherited from the cassandra-parent when adding these dependencies. So if the cassandra project updates the {metrics.version} version of the dropwizard metrics library and a user wants to update the cassandra-parent accordingly, all dropwizard metrics extensions will be the same version and there will be no conflict between them.

But maybe I'm missing something, so I don't have anything against using the `4.2.19` directly.

Reply:
transitive dependencies in maven resolve this for us.

Reply:
Fixed :-)